english/explore-outside/talk-child/smile/smile-to-child.md

```diff
@@ -0,0 +1 @@
+Even though the glaring red eyes are creepy, I decide to communicate with body languages. So I smile at the child reassuringly, extending my right hand towards him.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.